setuptools/config.py

```diff
@@ -262,22 +261,24 @@ def _parse_file(cls, value):
 
         spec = value[len(include_directive):]
         filepaths = (os.path.abspath(path.strip()) for path in spec.split(','))
+        return '\n'.join(
+            self._read_local_file(path)
+            for path in filepath
+            if os.path.isfile(path)
+        )
+
+    @staticmethod
+    def _read_local_file(filepath):
+        """
+        Read contents of filepath. Raise error if the file
+        isn't in the current directory.
+        """
+        if not filepath.startswith(os.getcwd()):
+            raise DistutilsOptionError(
+                '`file:` directive can not access %s' % filepath)
```

Comment:
I would prefer if `file:` directive name part was rendered from some reusable constant in this message.

Reply:
Yes, perhaps. This change makes that more difficult.

Reply:
> [...] from some reusable constant [...]

Mind Occam's razor.